item-nbt-api/src/main/java/de/tr7zw/changeme/nbtapi/NBTItem.java

```diff
@@ -16,18 +15,29 @@
 	private ItemStack bukkitItem;
 
 	/**
-	 * Constructor for NBTItems. The ItemStack will be cloned!
+	 * Constructor for NBTItems. The ItemStack will be cloned!<br/>
+	 * Use {@link NBTItem#getItem()} to get the changed ItemStack
 	 * 
 	 * @param item
```

Comment:
here

Reply:
[ERROR] /home/runner/work/Item-NBT-API/Item-NBT-API/item-nbt-api/src/main/java/de/tr7zw/changeme/nbtapi/NBTItem.java:18: error: self-closing element not allowed
[ERROR] 	 * Constructor for NBTItems. The ItemStack will be cloned!<br/>

Thats why the build is failing. Not adding descriptions to parms is just a warning, not an error (for now).

Reply:
Should also add a few tests for non cloned ItemStacks. I'm really unsure how broken this may end up when blindly working with ItemStacks that are in inventories, itemframes, on the ground etc.